### DISCUSSION TITLE
How do you create an OWL file and build an ontology from it?

### DISCUSSION THREAD

**Discussion by JisooSeo** [JisooSeo/101-WG-Discussions#7] [Q&A]
How do you create an OWL file and build an ontology from it?

**Comment by JisooSeo** [comment]
1. Go to Ontobee or OLS to look up terms and find the IRIs for the terms.

2. On Notepad, list all the terms in the form of 

                IRI # canine tooth

where “#” is used to comment on the IRI meaning “ignore what comes after #”

3. Name the text file “uberon_terms.txt”

4. Use makefile to make “uberon_import.owl”

5. Pull up the owl file on Github (desktop) and see that the newly added terms are in green. If you commit, then the change becomes part of the repository. 

6. Rebuild ontology at Protégé.

**Reply by JisooSeo** [reply]
Please correct or modify this!

**Comment by johnbeve** [comment]
I wouldn't mark this as answered quite yet ;)

**Reply by JisooSeo** [reply]
Thank you for your comment! I was using it as a "placeholder" answer but I should develop the answer.
